composer.json

```diff
@@ -20,7 +20,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.4 || ~8.0.0",
         "laminas/laminas-eventmanager": "^3.3"
     },
     "require-dev": {
```

Comment:
I think you miss to add psalm in `require-dev` section

Reply:
I did not introduce psalm at all, so far :D

Reply:
You have added some psalm annotation, maybe its a good start to introduce it to CI ?

Reply:
Yes, that is part of #45 - I generally add psalm annotations anyway, as they provide much richer type information, and I have psalm running with the LSP plugin in my IDE :+1:

Reply:
Ok :+1: